wiki/en/Running-a-Server.md

```diff
@@ -83,69 +66,34 @@ If you want to run a number of Servers, possibly also behind a firewall or on a
-1. Check all is well with `systemctl status jamulus-headless` (hit `q` to get back to the command prompt).
-
-### Configuration
-
-You can control Jamulus with the `systemctl` command. For example, to stop the Server cleanly:
-
-`sudo systemctl stop jamulus-headless`
-
-#### Running in Registered mode
-
-The following minimum setup is required to [run a Registered Server](Running-a-Server#server-types):
+Many people attribute problems to the Server that are in fact problems with the _Client_. Much depends on the Clients' hard/software setup, the networks that _they_ are on, and whether they are sticking to [Rule Number One](Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together).
 
-~~~
-jamulus --nogui --server \
-        --directoryserver genreServer:port \
-        --serverinfo "yourServerName;yourCity;[country ID]"
-~~~
+**_The capability of the Server itself (and the network it's on) is NOT the main determinant of the quality of a Jamulus session!_**
 
-**Note**: Semicolon and newline characters are not allowed in `yourServerName` and `yourCity` within the `--serverinfo` argument
+If you plan to be playing regularly with the same people, **you are strongly advised** to first make sure that each member of the group is set up to use Jamulus properly. Do this by finding a Public Server with a reasonable ping time for all of you (20 ms or less perhaps), all connect to that and work to fix any individual issues (verifying that they can [follow Rule Number One](Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together) in particular). Use the solo technique above to prevent being interrupted if needed.
 
-To register with one of the Directories built into the Jamulus Client, replace `genreServer:port` in the example above with one of the following options:
+Once any issues with musicians have been solved in this way, you can then investigate hosting your own Server either at home or on a cloud host such as Amazon, which may result in better latency than Servers run at home.
 
+### Bandwidth – do you have enough?
 
-| Genre |   Server Address           |
-|-----------|------------------|
-|**Any Genre 1** |`anygenre1.jamulus.io:22124`|
-|**Any Genre 2** |`anygenre2.jamulus.io:22224`|
-|**Any Genre 3** |`anygenre3.jamulus.io:22624`|
-|**Genre Rock** |`rock.jamulus.io:22424`|
-|**Genre Jazz** |`jazz.jamulus.io:22324`|
-|**Genre Classical/Folk** |`classical.jamulus.io:22524`|
-|**Genre Choral/Barbershop** |`choral.jamulus.io:22724`|
+Unless you plan on hosting more than about 5 players on a slower-speed home connection (eg 10 Mbit/s down and 1 Mbit/s up), then you are unlikely to run out of bandwidth. You can read more about network requirements at [different quality settings here](Server-Bandwidth).
 
-You can also specify a [Custom Directory](#3-custom-directory) in the same way from the command line, providing the Server Address in the same format.
 
-#### Running as a Directory
+## Setting up a server
```

Comment:
So when you click the link to "Server Installation page", you are given the impression that all servers are headless. Not sure that's a good idea.